src/index.ts

```diff
@@ -225,4 +227,23 @@ export default class SwupFormsPlugin extends Plugin {
 			this.specialKeys[event.key] = false;
 		}
 	};
+
+	/**
+	 * Handles visits triggered by forms matching [data-swup-inline-form]
+	 */
+	handleInlineForms: Handler<'visit:start'> = (visit) => {
+		const { el } = visit.trigger;
+		if (!el?.matches('form[data-swup-inline-form]')) return;
```

Comment:
I'd say we should make the selector configurable, just like the form selector itself. This is super handy for the astro integration, where we've set it to just `form` and we could add a global option there as well to rewrite the inline form selector.